lib/node_modules/@stdlib/math/base/special/gamma-lanczos-sum/scripts/evalrational.js

```diff
@@ -24,10 +24,15 @@
 // MODULES //
 
 var resolve = require( 'path' ).resolve;
+var readFileSync = require( '@stdlib/fs/read-file' ).sync;
 var writeFileSync = require( '@stdlib/fs/write-file' ).sync;
 var currentYear = require( '@stdlib/time/current-year' );
 var licenseHeader = require( '@stdlib/_tools/licenses/header' );
 var compile = require( '@stdlib/math/base/tools/evalrational-compile' );
+var compileC = require( '@stdlib/math/base/tools/evalpoly-compile-c' );
```

Comment:
Ah, I see. You need to use `evalrational-compile-c`. This will remove the need for `P1` and `Q1`, etc. You'll need to update here and in the C file.

Reply:
Thanks for pointing this out, I will make the suggested changes.